packages/config/src/forkConfig/index.ts

```diff
@@ -66,7 +66,7 @@ export function createForkConfig(config: ChainConfig): ForkConfig {
 
   /** Forks in order order of occurence, `phase0` first */
   // Note: Downstream code relies on proper ordering.
-  const forks = {phase0, altair, bellatrix, capella, deneb, verge};
+  const forks = {phase0, altair, bellatrix, capella, verge, deneb};
```

Comment:
The fork order could be dynamically derived from FORK_EPOCH_* values to prevent having to decide here which fork comes first